Cubical/Data/Sigma/Properties.agda

```diff
@@ -30,6 +30,9 @@ private
     B B' : (a : A) → Type ℓ
     C : (a : A) (b : B a) → Type ℓ
 
+mapʳ : (∀ {a} → B a → B' a) → Σ A B → Σ A B'
+mapʳ f (a , b) = (a , f b)
```

Comment:
Should we have map^l as well?

Reply:
Added for the case where B is constant, I'm can't imagine a map^l that is more useful than just using an eliminator in the case where is B is non-constant.